### DISCUSSION TITLE
Could not connect to X session manager: Could not open network socket

### DISCUSSION THREAD

**Discussion by zohercheddad** [canonical/steam-snap#232] [General]
```
os_release: 
    name:               "Ubuntu"
    version:            "22.04.2 LTS (Jammy Jellyfish)"
snap_info: 
    steam_revision:     136
    snapd_revision:     19122
lspci: 
    26:00.0:            Advanced Micro Devices, Inc. [AMD/ATI] Raven Ridge [Radeon Vega Series / Radeon Vega Mobile Series] (rev c8)
glxinfo: 
    gpu:                RAVEN (raven, LLVM 15.0.7, DRM 3.47, 5.19.0-41-generic)
    gpu_version:        4.6 (Core Profile) Mesa 23.0.3 - kisak-mesa PPA
lscpu: 
    model_name:         AMD Ryzen 3 2200G with Radeon Vega Graphics
xdg_current_desktop:    ubuntu:GNOME
desktop_session:        ubuntu
```

**Comment by zohercheddad** [comment]
Could not connect to X session manager: Could not open network socket
Could not connect to X session manager: Could not open network socket
Could not connect to X session manager: Could not open network socket
Could not connect to X session manager: Could not open network socket
ExecCommandLine: "'/home/zozo/snap/steam/common/.local/share/Steam/ubuntu12_32/steam' '-no-cef-sandbox'"
System startup time: 4.25 seconds
BuildCompleteAppOverviewChange: 156 apps
RegisterForAppOverview 1: 2ms
RegisterForAppOverview 2: 3ms
flock /usr/share/fonts/truetype/liberation2/LiberationSans-Regular.ttf LOCK_SH failed. errno = 13Could not connect to X session manager: Could not open network socket
Could not connect to X session manager: Could not open network socket
flock /usr/share/fonts/truetype/liberation2/LiberationSans-Regular.ttf LOCK_SH failed. errno = 13Steam: An X Error occurred
X Error:  BadWindow
Request Major code 20
ResourceID 0x0
Error Serial #9
xerror_handler: X failed, continuing
Steam: An X Error occurred
X Error:  BadWindow
Request Major code 20
ResourceID 0x0
Error Serial #9
xerror_handler: X failed, continuing
Steam: An X Error occurred
X Error:  BadWindow
Request Major code 20
ResourceID 0x0
Error Serial #9
xerror_handler: X failed, continuing
Installing breakpad exception handler for appid(steam)/version(1682708537)/tid(10533)
Installing breakpad exception handler for appid(steam)/version(1682708537)/tid(10534)
Warning: failed to set thread priority: set failed for -10: -1: setpriority() failed
Warning: failed to set thread priority: set failed for -10: -1: setpriority() failed
Steam: An X Error occurred
X Error:  BadWindow
Request Major code 20
ResourceID 0x0
Error Serial #9
xerror_handler: X failed, continuing

**Comment by ashuntu** [comment]
Is this in reference to a specific issue you are facing? Steam tends to log a lot of things that are harmless.

Also, the Game Reports category is specifically for reporting the working state of a specific game, so I've moved this to General.
